references/cli/cluster.go

```diff
@@ -71,10 +71,10 @@ const (
 func ClusterCommandGroup(c common.Args, ioStreams cmdutil.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "cluster",
-		Short: "Manage Clusters",
-		Long:  "Manage Clusters",
+		Short: "Manage Kubernetes Clusters",
+		Long:  "Manage Kubernetes Clusters",
 		Annotations: map[string]string{
-			types.TagCommandType: types.TypeSystem,
```

Comment:
Why Cluster Command belongs to CD?

Reply:
It belongs to the CD scenario for Deliver multi-cluster Application